version.go

```diff
@@ -29,8 +29,18 @@ const (
 	// appPreRelease MUST only contain characters from semanticAlphabet per
 	// the semantic versioning spec.
 	appPreRelease = "beta"
+
+	// defaultAgentName is the default name of the software that is added as
+	// the first part of the user agent string.
+	defaultAgentName = "loopd"
 )
 
+// AgentName stores the name of the software that is added as the first part of
+// the user agent string. This defaults to the value "loopd" when being run as
+// a standalone component but can be overwritten by LiT for example when loopd
+// is integrated into the UI.
+var AgentName = defaultAgentName
```

Comment:
When you say "can be overwritten..." how do you mean exactly? Should this be set trough the RPC interface when integrated?

Reply:
We add `loopd` as a library in LiT and can simply set this global package-level variable. No need to add another RPC interface.

Reply:
I see, thanks for the explanation.